tcl/windows/pgn.tcl

```diff
@@ -104,7 +104,7 @@ namespace eval pgn {
     $w.menu.opt add checkbutton -label PgnOptStripMarks \
         -variable ::pgn::stripMarks -command {updateBoard -pgn}
     $w.menu.opt add checkbutton -label PgnOptBoldMainLine \
-        -variable ::pgn::boldMainLine -command {updateBoard -pgn}
+        -variable ::pgn::boldMainLine -command {::pgn::Refresh 1}
```

Comment:
Probably just invoking configure with the right font should work and invoking ::pgn::Refresh may non be necessary.